src/components/IndividualMovie.js

```diff
@@ -12,16 +12,16 @@ const IndividualMovie = props => {
           <h2 className="movie-title">{props.movie.title}</h2>
           <h3 className="movie-tagline">{props.movie.tagline}</h3>
         </section>
-          <p className="movie-overview"><span className="bold">Overview:</span> {props.movie.overview}</p>
+          <p className="movie-overview"><span className="bold">Overview:</span> {props.movie.overview === '' ? <p>No overview available for this movie at this time! Please submit your version of the movie overview <a href="http://www.turing.edu/contact" target="_blank">here</a></p> : props.movie.overview}</p>
```

Comment:
Good practice for conditional rendering - Is there a way to use a helper function of some sort to make it cleaner to read?